apps/web/cypress/tests/notifications-editor.spec.ts

```diff
@@ -266,7 +311,7 @@ describe('Notifications Creator', function () {
       cy.getByTestId('groupSelector').should('have.value', 'New Test Category');
     });
 
-    it('should edit notification', function () {
+    it.skip('should edit notification', function () {
```

Comment:
this test was breaking, I'm not sure why... check the comment below

Reply:
I will take a look at it once merge, do you mind opening an issue for that?

Reply:
sure will do that!

Reply:
@scopsy the ticket https://github.com/novuhq/novu/issues/948